index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="index.css">
+    <title>Hello</title>
+</head>
+<body>
+    <h1>Hello Microverse!</h1>
+</body>
```

Comment:
- [OPTIONAL] I would recommend you to add a class name to your `h1` element and use that class name to style its content in the css file. It is marked optional because, it was not mentioned by the previous reviewer.